BlueprintUICommonControls/Sources/ScrollView.swift

```diff
@@ -145,6 +164,10 @@ extension ScrollView {
 fileprivate final class ScrollerWrapperView: UIView {
     
     let scrollView = UIScrollView()
+    let keyboardObserver = KeyboardObserver()
+    
+    /// The current `ScrollView` state we represent.
+    private var representedElement : ScrollView
```

Comment:
Can we just hang onto the values we need rather than the whole element? This is really tightly coupled.

Reply:
I went back and forth on this... Started with the individual values, and then went back to this since these types seem inherently tightly coupled, and it make the code easier to read/write, IMO.